Plugin/homograph.ps1

```diff
@@ -12,7 +12,7 @@ $pluginPaths = @()
 
 $parameters = @{
     packageName     = "EdFi.Suite3.Ods.Extensions.Homograph"
-    packageVersion  = "5.2.0-b10505"
+    packageVersion  = "5.2.0-b10513"
```

Comment:
These build numbers will need to be updated again before merge, particularly since the projects will need to be updated to embed the _assemblyMetadata.json_ file as an embedded resource in the assembly, per acceptance criteria on [ODS-4664](https://tracker.ed-fi.org/browse/ODS-4664).